www/tests/test_app.py

```diff
@@ -1,4 +1,11 @@
 import app
 
-def test_dummy():
+def test_sensor_init():
     assert len(app.binary_sensors) > 0
+
+def test_get_local_ip():
```

Comment:
Yeah, I'll try it out!
The other challenge with heartbeat_thread is that it is an infinite loop. Any ideas on how to get around that problem?